js/src/d3.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var d3Selection = require('d3-selection');
+var d3Xhr = require('d3-xhr');
+var d3Array = require('d3-array');
+var scale = require('d3-scale');
+var d3Format = require('d3-format');
+
+const functor = function(x) {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).